content/news/xxxx-xx-xx-mixxx-2-3-1-released.md

```diff
@@ -0,0 +1,18 @@
+title: Mixxx 2.3.1 released
+authors: Daniel Schürmann
+tags: 2.3, 2.3.1, release announcement
+status: draft
+comments: yes
+
+We are happy to announce first minor release of [Mixxx 2.3.1](https://github.com/mixxxdj/mixxx/releases/tag/2.3.1)!
+You will find the download links [here]({filename}/pages/download.md#stable) on Windows, macOS and Linux.
+
+This release comes with new and updated controller mappings, fixes packaging issues and includes some bugfixes.
+
+Mixxx now includes mappings for the [Numark DJ2GO2 Touch](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_dj2go2_touch.html) controller and the [Numark Mixtrack Pro FX](https://manual.mixxx.org/2.3/en/hardware/controllers/numark_mixtrack_pro_fx.html) controller. The [Behringer DDM4000](https://manual.mixxx.org/2.3/en/hardware/controllers/behringer_ddm4000.html), the [Hercules Inpulse 300](https://manual.mixxx.org/2.3/en/hardware/controllers/hercules_djcontrol_inpulse_300.html) mapping, the [Denon MC7000](https://manual.mixxx.org/2.3/en/hardware/controllers/denon_mc7000.html) mapping and the [Traktor S3](https://manual.mixxx.org/2.3/en/hardware/controllers/native_instruments_traktor_kontrol_s3.html) mapping have been improved with Mixxx 2.3.
+
+Among other minor issue this release fixes the random skipping of tracks and the wrong track entries in the History list when using Auto-DJ, regressions since Mixxx 2.3.0.
+
+For the full list of changes, have a look at the [changelog](https://manual.mixxx.org/2.3/en/chapters/appendix/changelog.html) and the [2.3.1 milestone on Launchpad](https://launchpad.net/mixxx/+milestone/2.3.1).
+
+In case your are a Linux package maintainer please double check the [Packaging](https://manual.mixxx.org/2.3/en/chapters/appendix/changelog.html#packaging) chapter in the changelog and modify your packaging script if necessary.
```

Comment:
```suggestion
In case you're a Linux package maintainer please double check the [Packaging section](https://github.com/mixxxdj/mixxx/blob/2.3.1/CHANGELOG.md#packaging) in the changelog and modify your packaging script if necessary.
```